w-books/src/App/screens/Login/index.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import { withRouter } from 'react-router';
+import './styles.css';
+import Authenticate from '../../../services/AuthService.js'
+import wlogo from '../../../assets/Header/wbooks_logo.svg';
+
+class Login extends Component {
+  state = {email: "", password: ""}
+  inputEmail = (e) => {
+    this.setState({email: e.target.value})
+  }
+  inputPassword = (e) => {
+    this.setState({password: e.target.value})
+  }
+  submit = (e) => {
+    Authenticate({email: this.state.email,
+                  password: this.state.password })
+                  .then(() => {
+                    this.props.history.push('/dashboard')
+                  })
+                  .catch((error) => {
+                    alert (error);
+                  });
+  }
+  validateEmail = () => {
+    return (this.state.email.length > 0 && this.state.email.includes("@"))
+  }
+  validatePassword = () => {
+    if (this.state.password.length >= 8 && this.state.password.length <= 52) {
+      const regLetters = new RegExp("[^a-z|^A-Z]");
+      const regNumbers = new RegExp("[^0-9]");
+      return regLetters.test(this.state.password) && regNumbers.test(this.state.password);
+    }
+    return false
+  }
+  render() {
+    const inputValid = this.validateEmail() && this.validatePassword()
+    return (
```

Comment:
`<Login inputEmail={this.inputEmail} inputPassword={this.inputPassword} onSubmit={this.submit} />`

Reply:
smart component on `login/index.js` and dumb component in `login/layout.js`